<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1903</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1903</width>
    <height>1050</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1903</width>
    <height>1050</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(240, 255, 165);
background-color: rgb(207, 255, 189);
color: rgb(234, 255, 187);
color: rgb(255, 241, 205);
color: rgb(255, 0, 4);
color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
background-color: rgb(255, 240, 165);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тема 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тема 2 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>720</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тема 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn5">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>220</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тема 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1_ab">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>740</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Немного о приложении</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>90</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Качественные реации</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_to">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>750</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тесты ОГЭ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_te">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>750</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тесты ЕГЭ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>480</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
color: rgb(255, 0, 4);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Тема 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>180</y>
      <width>501</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Users/Lenovo/Downloads/project_image2.jpg</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
